std_msgs/Header header

# Frame number which is associated to images by timestamp in header
int32 frame_number
# Published frames per second
float64 publish_rate
# Render pass of images
string render_pass
# Name of published sequence directory
string sequence_name
